internal/utils/files/main.go

```diff
@@ -3,15 +3,15 @@ package files
 import (
```

Comment:
I must have missed this file in a previous PR, I see it's not new - but could you add a header to why we have this file? I am confused about why it is needed. If it is only needed for the fixtures lookup, consider moving it to that file.

Reply:
It has been a part of `probe_test` to read out the jsonnet files, but after 5dcbf7324ee258c0fc0b99cad87cd517b6bc26a7 this has not be needed anymore. I have no special feelings regarding to keep it there or move it. If you prefer to move it all to `internal/utils/fixture` that's fine for me.